src/react/index.ts

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import {provideReactWrapper} from '@microsoft/fast-react-wrapper';
+import {provideFASTDesignSystem} from '@microsoft/fast-components';
```

Comment:
The toolkit actually exports its own `provideVSCodeDesignsystem` from `src/index.ts` that should be used here instead of the FAST design system.

```typescript
import {provideVSCodeDesignSystem} from '../index';
```

Reply:
Resolved by 82e710a8991eebce74fdb702c03f81171c6b684d